### PR TITLE
Fixed: Updates to System > More Info

### PR DESCRIPTION
**Pull request by bakerboy448**
#### Database Migration
NO

#### Description
Frontend More Info updates to be consistent across Lidarr/Readarr/Radarr

#### Todos
-  Tests-N/A
- [X] Wiki Updates

#### Issues Fixed or Closed by this PR

*

### CODE REVIEW

File: frontend/src/System/Status/MoreInfo/MoreInfo.js [hunk 1]
```diff
@@ -23,24 +23,38 @@ class MoreInfo extends Component {
...
-            <Link to="https://wiki.servarr.com/Radarr">wiki.servarr.com/Radarr</Link>
+            <Link to="https://discord.gg/r5wJPt9">Radarr</Link>
           </DescriptionListItemDescription>
 
           <DescriptionListItemTitle>
             {translate('Donations')}
           </DescriptionListItemTitle>
           <DescriptionListItemDescription>
-            <Link to="https://opencollective.com/radarr">opencollective.com/radarr</Link>
+            <Link to="https://opencollective.com/radarr">Donate to Radarr</Link>
```
Comment (austinwbest) [inline frontend/src/System/Status/MoreInfo/MoreInfo.js:50]: The title is translated, now that the link text is changed to not be an actual link it should be translated as well.

File: frontend/src/System/Status/MoreInfo/MoreInfo.js [hunk 2]
```diff
@@ -23,24 +23,38 @@ class MoreInfo extends Component {
...
+          </DescriptionListItemDescription>
+
+          <DescriptionListItemTitle>
+            {translate('DonationsSonarr')}
+          </DescriptionListItemTitle>
+          <DescriptionListItemDescription>
+            <Link to="https://sonarr.tv/donate">Donate to Sonarr</Link>
```
Comment (austinwbest) [inline frontend/src/System/Status/MoreInfo/MoreInfo.js:57]: The title is translated, link text should be translated as well.

**Review by austinwbest — CHANGES_REQUESTED** [review]

File: frontend/src/System/Status/MoreInfo/MoreInfo.js [hunk 3]
```diff
@@ -23,24 +23,38 @@ class MoreInfo extends Component {
...
+          <DescriptionListItemDescription>
+            <Link to="https://www.reddit.com/r/Radarr/">Radarr</Link>
+          </DescriptionListItemDescription>
+
+          <DescriptionListItemTitle>
+            {translate('Discord')}
           </DescriptionListItemTitle>
           <DescriptionListItemDescription>
-            <Link to="https://wiki.servarr.com/Radarr">wiki.servarr.com/Radarr</Link>
+            <Link to="https://discord.gg/r5wJPt9">Radarr</Link>
```
Comment (Qstick) [inline frontend/src/System/Status/MoreInfo/MoreInfo.js:43]: Might as well change this while you are here

**Comment by Qstick** [comment]
Make a section with icons for donations
image

**Comment by bakerboy448** [comment]
@Qstick punting on the icons; even with the bit of discussion the other day, still over my head.  seems like a good future idea tho, maybe FR it.

this has been rebased though

**Review by austinwbest — APPROVED** [review]

**Comment by RobinDadswell** [comment]
@bakerboy448  want to sqaush these?

**Review by Qstick — APPROVED** [review]

**Review by Qstick — CHANGES_REQUESTED** [review]
Good call on the squash

**Review by Qstick — APPROVED** [review]

**Pull request merged**